clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -37,56 +37,73 @@
   ],
   "icon": "resources/logo.png",
   "main": "./dist/extension.js",
+  "browser": "./dist/web/extension.js",
+  "capabilities": {
+    "virtualWorkspaces": {
+      "supported": "limited",
+      "description": "In virtual workspaces, only syntax highlighting is supported."
+    }
+  },
   "contributes": {
     "commands": [
       {
         "command": "cobol-lsp.cpy-manager.fetch-copybook",
         "category": "COBOL",
-        "title": "Fetch Copybook"
+        "title": "Fetch Copybook",
+        "when": "!virtualWorkspace"
```

Comment:
There is a variable `isWeb` that more appropriately represents the condition.

Reply:
The "!virtualWorkspace" was taken from Microsoft's documentation: https://code.visualstudio.com/api/extension-guides/virtual-workspaces#disable-functionality-when-a-virtual-workspace-is-opened

Reply:
virtual workspace can be opened even in the desktop vscode.

Reply:
Changed to use "!isWeb" in later commit.